README.md

```diff
@@ -1 +1,39 @@
-# awesome-books
\ No newline at end of file
+# Awesome Books
+
+> This my Awesome Books web app
+
```

Comment:
- [OPTIONAL] I think it would be better to add What your application does? 👍

> Your project description is an extremely important aspect of your project. A well-crafted description allows you to show off your work to other developers as well as potential employers.